internal/verifier/verification_task.go

```diff
@@ -59,11 +59,19 @@ type VerificationTask struct {
 	Type       verificationTaskType   `bson:"type"`
 	Status     verificationTaskStatus `bson:"status"`
 	Generation int                    `bson:"generation"`
-	Ids        []interface{}          `bson:"_ids"`
+
+	// For failed tasks, this stores the document IDs missing on
+	// one cluster or the other.
+	Ids []interface{} `bson:"_ids"`
+
 	// Deprecated: VerificationTask ID field is ignored by the verifier.
-	ID          int                  `bson:"id"`
-	FailedDocs  []VerificationResult `bson:"failed_docs,omitempty"`
-	QueryFilter QueryFilter          `bson:"query_filter"          json:"query_filter"`
+	ID int `bson:"id"`
+
+	// For failed tasks, this stores details on documents that exist on
+	// both clusters but don’t match.
+	FailedDocs []VerificationResult `bson:"failed_docs,omitempty"`
```

Comment:
Would it make sense to rename this to "MismatchedDocs"?

Reply:
Possibly, but the verifier kind of overloads the distinction between “failed” and “mismatched”. I’m not sure that this doesn’t get alternately used in both cases. :-P